randovania/games/prime1/logic_database/header.txt

```diff
@@ -82,6 +82,13 @@ Dock Weaknesses
       No lock
 
 
+  * Power Door
```

Comment:
IMO `Power Beam Door` sounds nicer

Reply:
yes, but it doesn't fit with all the others `wave door, plasma door, ice door`

Reply:
The existing ones should be renamed too IMO